
int a

try {
    a = 10 / 0  // Error: divisiÃ³n por cero
} catch (error) {
    a = 5 // se atrapa el error y se ejecuta el bloque
} finally {
    int b = a
}

bool b

try {
    b = 10 > 0  // Error: error logico de comparacion
} catch (error) {
    b = true // se atrapa el error y se ejecuta el bloque
}
